### PR TITLE
fix(resource-status/Graph): Fix base 1024 formatting

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This fixes the rounding problem on base 1024 graph units.

### CODE REVIEW

File: www/front_src/src/Resources/Graph/Performance/formatMetricValue/index.test.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ type TestCase = [number, string, 1000 | 1024, string];
 describe(formatMetricValue, () => {
   const cases: Array<TestCase> = [
     [218857269, '', 1000, '218.86m'],
-    [218857269, '', 1024, '219M'],
+    [218857269, '', 1024, '208.72M'],
     [0.12232323445, '', 1000, '0.12'],
     [1024, 'B', 1000, '1K'],
```
Comment (kduret) [inline www/front_src/src/Resources/Graph/Performance/formatMetricValue/index.test.ts:10]: this test should breaks

Comment (kduret) [reply www/front_src/src/Resources/Graph/Performance/formatMetricValue/index.test.ts:10]: my bad, B unit forces base to 1024

**Review by kduret — DISMISSED** [review]

File: www/front_src/src/Resources/Graph/Performance/formatMetricValue/index.ts [hunk 2]
```diff
@@ -13,9 +13,11 @@ const formatMetricValue = ({ value, unit, base = 1000 }): string => {
 
   const base1024 = base2Units.includes(unit) || base === 1024;
 
-  const format = base1024 ? '0b' : '0.[00]a';
+  const formatSuffx = base1024 ? ' ib' : 'a';
```
Comment (Thebarda) [inline www/front_src/src/Resources/Graph/Performance/formatMetricValue/index.ts:16]:
```suggestion
  const formatSuffix = base1024 ? ' ib' : 'a';
```

File: www/front_src/src/Resources/Graph/Performance/formatMetricValue/index.ts [hunk 3]
```diff
@@ -13,9 +13,11 @@ const formatMetricValue = ({ value, unit, base = 1000 }): string => {
...
 
-  return numeral(value).format(format).replace('B', '');
+  return numeral(value)
+    .format(`0.[00]${formatSuffx}`)
```
Comment (Thebarda) [inline www/front_src/src/Resources/Graph/Performance/formatMetricValue/index.ts:19]:
```suggestion
    .format(`0.[00]${formatSuffix}`)
```

**Review by kduret — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**